tests/e2e/forms/submit-photo-upload-form.spec.js

```diff
@@ -20,47 +21,27 @@ const userContactDoc = {
 };
 
 describe('Submit Photo Upload form', () => {
-  beforeAll(done => {
-    Promise.resolve()
-      .then(() => photoUpload.configureForm(userContactDoc, done))
-      .catch(done.fail);
+  beforeAll(async () => {
+    await photoUpload.configureFormNative(userContactDoc);
   });
 
-  afterEach(done => {
-    utils.resetBrowser();
-    done();
+  afterEach(async () => {
+    await utils.resetBrowser();
   });
 
-  afterAll(utils.afterEach);
+  afterAll(async () => { await utils.afterEachNative();});
 
-  it('upload photo', () => {
-    common.goToReports();
-
-    // select form
-    const addButton = element(
-      by.css('.action-container .general-actions:not(.ng-hide) .fa-plus')
-    );
-    helper.waitElementToPresent(addButton);
-    helper.clickElement(addButton);
-    element(
-      by.css(
-        '.action-container .general-actions .dropup.open .dropdown-menu li:first-child a'
-      )
-    ).click();
-    helper.waitElementToPresent(
-      element(by.css('#photo-upload input[type=file]'))
-    );
-    element(by.css('#photo-upload input[type=file]')).sendKeys(
+  it('upload photo', async () => {
+    await  common.goToReports();
```

Comment:
Buy-one-get-one-free spaces! ;)
```suggestion
    await common.goToReports();
```